src/assert.c

```diff
@@ -21,7 +21,7 @@ void assert(
 		#ifdef DEVEL
 		{
 			PRINTF("Assertion failed %s\n", msgStr);
-			ui_displayConfirm("Assertion failed", msgStr, NULL, NULL);
+			ui_displayPrompt("Assertion failed", msgStr, NULL, NULL);
```

Comment:
I guess we can move this to ui_displayPaginatedText, there are anyway just null handlers